### PR TITLE
Add remaining gh actions for cm submission generation

### PR DESCRIPTION
**Pull request by anandhu-eng**

### CODE REVIEW

File: .github/workflows/test-cm-based-submission-generation.yml [hunk 1]
```diff
@@ -73,5 +80,17 @@ jobs:
         # Dynamically set the log group to simulate a dynamic step name
         echo "::group::$description"
         cm ${{ matrix.action }} script --tags=generate,inference,submission --adr.submission-checker-src.tags=_branch.dev --clean --preprocess_submission=yes --results_dir=$PWD/submission_generation_tests/${{ matrix.case }}/ --run-checker --submitter=MLCommons --tar=yes --env.CM_TAR_OUTFILE=submission.tar.gz --division=${{ matrix.division }} --env.CM_DETERMINE_MEMORY_CONFIGURATION=yes --quiet $extra_run_args
+        
+        if [[ "${{ matrix.case }}" -eq 5 || "${{ matrix.case }}" -eq 6 ]]; then
+          # For cases 5 and 6, exit status should be 0 if cm command fails, 1 if it succeeds
+          if [[ $exit_status -ne 0 ]]; then
```
Comment (arjunsuresh) [inline .github/workflows/test-cm-based-submission-generation.yml:86]: How's exit_status populated?

File: .github/workflows/test-cm-based-submission-generation.yml [hunk 2]
```diff
@@ -62,6 +79,19 @@ jobs:
         fi
         # Dynamically set the log group to simulate a dynamic step name
         echo "::group::$description"
-        cm ${{ matrix.action }} script --tags=generate,inference,submission --adr.submission-checker-src.tags=_branch.dev --clean --preprocess_submission=yes --results_dir=$PWD/submission_generation_tests/${{ matrix.case }}/ --run-checker --submitter=MLCommons --tar=yes --env.CM_TAR_OUTFILE=submission.tar.gz --division=${{ matrix.division }} --env.CM_DETERMINE_MEMORY_CONFIGURATION=yes --quiet $extra_run_args
+        eval cm ${{ matrix.action }} script --tags=generate,inference,submission --adr.submission-checker-src.tags=_branch.dev --clean --preprocess_submission=yes --results_dir=$PWD/submission_generation_tests/${{ matrix.case }}/ --run-checker --submitter=MLCommons --tar=yes --env.CM_TAR_OUTFILE=submission.tar.gz --division=${{ matrix.division }} --env.CM_DETERMINE_MEMORY_CONFIGURATION=yes --quiet $extra_run_args
+        exit_status=$?
+        echo "Exit status for the job $description $exit_status"
+        if [[ "${{ matrix.case }}" == "case-5" || "${{ matrix.case }}" == "case-6" ]]; then
```
Comment (arjunsuresh) [inline .github/workflows/test-cm-based-submission-generation.yml:85]: ${{ matrix.case }} cannot be put inside bash if. We need to explictly assign this to a bash variable

Comment (anandhu-eng) [reply .github/workflows/test-cm-based-submission-generation.yml:85]: Thanks @arjunsuresh , let me try with this lead

Comment (anandhu-eng) [reply .github/workflows/test-cm-based-submission-generation.yml]: Hi @arjunsuresh , but [this](https://github.com/mlcommons/cm4mlops/blob/f50e302d67155672432b7bc12ab1db4e8785f866/.github/workflows/test-cm-based-submission-generation.yml#L61) works

Comment (arjunsuresh) [reply .github/workflows/test-cm-based-submission-generation.yml]: yes, I think both should work as the string is replaced by github action before it reaches bash.

**Pull request merged**